client/components/marketing-survey/gsuite-cancel-purchase-dialog/gsuite-cancellation-features.jsx

```diff
@@ -17,29 +17,43 @@ class GSuiteCancellationFeatures extends Component {
 		this.props.recordTracksEvent( 'calypso_purchases_gsuite_remove_purchase_learn_more_click' );
 	};
 
+	getTimePeriod = () => {
+		const { status, translate } = this.props;
+		return status === 'suspended'
+			? translate( 'immediately' )
+			: translate( 'in %(days)s days', { args: { days: '30' }, comment: 'Number of days' } );
+	};
+
 	render() {
 		const { purchase, translate } = this.props;
 		const { meta: domainName, productSlug } = purchase;
 
 		return (
 			<div className="gsuite-cancel-purchase-dialog__features">
 				<CardHeading tagName="h3" size={ 24 }>
-					{ translate( "Are you sure? Here's what you'll be missing:" ) }
+					{ translate( 'Are you sure?' ) }
 				</CardHeading>
 
 				<p>
 					{ translate(
-						'If you cancel and remove %(googleMailService)s from {{siteName/}} you will lose access to the following: ',
+						'If you cancel your subscription now, you will lose access to all of ' +
+							'your %(googleMailService)s features %(days)s. After that time, ' +
+							'you will need to purchase a new subscription with Google.',
 						{
 							args: {
 								googleMailService: getGoogleMailServiceFamily( productSlug ),
+								days: this.getTimePeriod(),
 							},
-							comment: '%(googleMailService)s can be either "G Suite" or "Google Workspace"',
-							components: { siteName: <em>{ domainName }</em> },
+							comment:
+								'%(googleMailService)s can be either "G Suite" or "Google Workspace" and %(days)s might be "immediately" or "after 30 days"',
 						}
 					) }
 				</p>
 
+				<CardHeading tagName="h3" size={ 24 }>
+					{ translate( "Here’s what you'll be missing" ) }
```

Comment:
Can we add a trailing `:` to make it clear that there is a list after the text? (The previous text had this as well.)
```suggestion
					{ translate( "Here’s what you'll be missing:" ) }
```

Reply:
Ups 🙂